src/sys.c

```diff
@@ -292,6 +292,51 @@ JL_DLLEXPORT jl_value_t *jl_readuntil(ios_t *s, uint8_t delim, uint8_t str)
     return (jl_value_t*)a;
 }
 
+JL_DLLEXPORT jl_value_t *jl_readline(ios_t *s, uint8_t chomp)
```

Comment:
This is a lot of near-duplicate code. Could we add an argument to `jl_readuntil` instead? Similarly `ios_copyline` does not seem totally necessary.